cmd/fastly/main.go

```diff
@@ -39,11 +38,118 @@ func main() {
 		out            io.Writer = common.NewSyncWriter(os.Stdout)
 	)
 
+	// We have to manually handle the inclusion of the verbose flag here because
+	// Kingpin doesn't evaluate the provided arguments until app.Run which
+	// happens later in the file and yet we need to know if we should be printing
+	// output related to the application configuration file in this file.
+	var verboseOutput bool
+	for _, seg := range args {
+		if seg == "-v" || seg == "-verbose" || seg == "--verbose" {
```

Comment:
I don't think `-verbose` is valid?

```suggestion
		if seg == "-v" || seg == "--verbose" {
```